src/components/Editor/GeneralEditor.test.tsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-this-alias */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { GeneralEditor } from './GeneralEditor';
+import { VideoEditorToolbar } from './Editors/VideoEditorToolbar';
+
+global.document.execCommand = jest.fn((value) => value);
+
+const mockHtmlContentCange = jest.fn((value) => value);
+const mockTextContantChange = jest.fn((value) => value);
+jest.mock(
+  'quill-image-uploader',
+  () =>
+    function (quill, options) {
+      const self = this;
```

Comment:
why is this needed? 

Here's how to mock classes wiht jest

https://jestjs.io/docs/es6-class-mocks

Reply:
![imageUploaderError](https://user-images.githubusercontent.com/56695491/125650768-53626879-c4ee-4501-8357-7c633850accd.PNG)

Reply:
![imageUploaderSuccess](https://user-images.githubusercontent.com/56695491/125650828-da291eb4-561c-48b4-a5e5-42e5dd62e8eb.PNG)